CONTRIBUTING.md

```diff
@@ -0,0 +1,163 @@
+- [Get in touch](#get-in-touch)
+
+
+## Getting started
+
+### Decide what you want to work on
+
+If you are looking for something to work on, you can look through our [issues](https://github.com/cabotage/cabotage-app/issues) and pick some you like. We try to maintain a list of issues that should be suitable for first time contributions, they can be found tagged [`good first contribution`](https://github.com/cabotage/cabotage-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+If you’re still unsure, please contact us and we would help you to the best of our abilities.
+
+
+### Notify your interest
+
+Please let us know you want to work on it so we can avoid multiple people working on the same issue. You can leave a comment in the issue tagging the author to notify your interest to work on the issue.
+
+Before starting any large pull requests (like adding features or reworking the code), **please ask first**. If you don’t, you run the risk of investing a lot of time in something that the project’s developers might decide not to include.
+
+
+### Setting up the project in your local machine
+
+1. [Fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) the project, clone your fork, and configure the remotes:
+
+    ```sh
+    # Clone your fork of the repo into the current directory
+    git clone https://github.com/<your-username>/<repo-name>
+
+    # Navigate to the newly cloned directory
+    cd <repo-name>
+
+    # Assign the original repo to a remote called "upstream"
+    git remote add upstream https://github.com/cabotage/cabotage-app
+    ```
+
+2. Create a new topic branch to contain your feature, change, or fix:
+
+    ```sh
+    git checkout -b <topic-branch-name>
+    ```
+
+3. Write clear and meaningful git commit messages.
+    We use the imperative mood as a coding convention, which typically follows this structure:
+
+    ```sh
+    <imperative verb> <description>
+
+    [optional body]
+    ```
+
+    Separate subject from body with a blank line and use the body to explain what changed and why.
+
+    To find some examples, you can checkout the style of the commit messages on our [main branch](https://github.com/cabotage/cabotage-app/commits/main/).
+
+4. Make sure to update or add to the tests when appropriate. Run the appropriate testing suites to check that all tests pass after you’ve made changes.
+
+5. If you added or changed a feature, make sure to document it accordingly in the [README.md](./README.md) file, when appropriate.
+
+
+### Local setup and testing
+
+Make sure to add tests for any new features or improvements made to the code. Details on local setup and running tests can be found in the [README.md](./README.md) file.
+
+
+### Code formatting
+Before committing changes, you can lint and reformat your code using [tox](https://tox.wiki/). To install this tool on your local machine or a virtual environment:
+
+```sh
+pip install tox
+```
```

Comment:
This is also described in the readme file. Maybe add the `make lint` etc to readme and reference it? What do you think?

Reply:
Or merge the section with local setup and testing :)

Reply:
This was copied and removed from the readme so no duplications